src/Sentry.Unity/SentryUnityOptions.cs

```diff
@@ -160,6 +160,12 @@ public sealed class SentryUnityOptions : SentryOptions
     /// </summary>
     public bool AndroidNativeSupportEnabled { get; set; } = true;
 
+    /// <summary>
+    /// Whether the SDK should initialize the native SDK before the Unity game. This bakes the options at build-time into
+    /// the generated Gradle project
+    /// </summary>
```

Comment:
This option need to be documented. 

An idea: the paragraph explaining what "baking into" means, could be a a paragraph that u make an 'include' so u can add on the iOS as well as Android option (since there are two).

```suggestion
    /// </summary>
    /// <seealso href="https://docs.sentry.io/platforms/unity/configuration/options/#android-native-initialization-type"
```

Reply:
I've added additional remarks to the enum itself and to the `inittype` options. I'll add the link to our docs once they are updated.